log_drains.go

```diff
@@ -0,0 +1,71 @@
+package scalingo
+
+import (
+	"gopkg.in/errgo.v1"
+)
+
+type LogDrainsService interface {
+	LogDrainsList(app string) ([]LogDrain, error)
+	LogDrainAdd(app string, params LogDrainAddParams) (*LogDrain, error)
+	LogDrainRemove(app string, id string) error
+}
+
+var _ LogDrainsService = (*Client)(nil)
+
+type LogDrain struct {
+	AppID string `json:"app_id"`
+	URL   string `json:"url"`
+}
+
+type LogDrainsRes struct {
+	LogDrains []LogDrain
+}
+
+type LogDrainRes struct {
+	LogDrain LogDrain `json:"log_drain"`
+}
+
+func (c *Client) LogDrainsList(app string) ([]LogDrain, error) {
+	var logDrainsRes []LogDrain
+	err := c.ScalingoAPI().SubresourceList("apps", app, "log_drains", nil, &logDrainsRes)
+	if err != nil {
+		return nil, errgo.Mask(err)
```

Comment:
We don't use `errgo.Mask` anymore. Please use `errgo.Notef` instead.
```suggestion
		return nil, errgo.Notef(err, "fail to list the log drains")
```